docs/mindsdb-docs/docs/sql/tutorials/house-sales-forecasting.md

```diff
@@ -1,60 +1,158 @@
+    And [this guide](https://docs.mindsdb.com/sql/create/file/) explains how to upload a file to MindsDB.
 
+    Now, you can query the uploaded file as if it were a table.
+
+    ```sql
+    SELECT *
+    FROM files.house_sales
+    LIMIT 10;
+    ```
+
+!!! Warning "From now on, we will use the `files.house_sales` file as a table. Make sure you replace it with `#!sql example_db.demo_data.house_sales` if you use the `demo` database."
+
+### Understanding the Data
+
+We will use the house sales dataset where each row represents one real estate (house or unit). It tracks quarterly moving averages of house sales aggregated by type and amount of bedrooms in each listing. These quarterly moving averages will be predicted in the following sections of this tutorial.
+
+Below is the sample data stored in the house sales dataset.
+
+```sql
++----------+------+-----+--------+
+|saledate  |MA    |type |bedrooms|
++----------+------+-----+--------+
+|30/09/2007|441854|house|2       |
+|31/12/2007|441854|house|2       |
+|31/03/2008|441854|house|2       |
+|30/06/2016|430880|unit |2       |
+|30/09/2016|430654|unit |2       |
++----------+------+-----+--------+
 ```
-CREATE PREDICTOR 
-  mindsdb.home_sales_model
+
+Where:
+
+| Column                | Description                                                                                  | Data Type           | Usage   |
+| :-------------------- | :------------------------------------------------------------------------------------------- | ------------------- | ------- |
+| `saledate`            | The date of sale                                                                             | `date`              | Feature |
+| `MA`                  | Moving average of the historical median price of the house or unit                           | `integer`           | Label   |
+| `type`                | Type of property (either a house or a unit)                                                  | `character varying` | Feature |
+| `bedrooms`            | Number of bedrooms                                                                           | `integer`           | Feature |
+
+!!!Info "Labels and Features"
+
+    A **label** is the thing we're predicting — the y variable in simple linear regression.
+    A **feature** is an input variable — the x variable in simple linear regression.
+
+## Training a Predictor Via [`#!sql CREATE PREDICTOR`](/sql/create/predictor)
+
+Let's create and train your first machine learning predictor. For that, we are going to use the [`#!sql CREATE PREDICTOR`](/sql/create/predictor) syntax where we specify what sub-query to train `#!sql FROM` (features) and what we want to `#!sql PREDICT` (labels).
+
+```sql
+CREATE PREDICTOR mindsdb.house_sales_predictor
 FROM files
-  (SELECT * FROM HR_MA)
+  (SELECT * FROM house_sales)
 PREDICT MA
 ORDER BY saledate
 GROUP BY bedrooms, type
 -- as the target is quarterly, we will look back two years to forecast the next one
```

Comment:
Break the comments on multiple lines to prevent horizontal scroll.

Reply:
Done.